presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -890,4 +890,79 @@ CTE Materialization Properties
 
     The name of the catalog to be used for Common Table Expressions (CTE) and which provides custom partitioning for Common Table Expression (CTE) materialization.
     This setting specifies which catalog should be used for CTE materialization and for determining how to partition the materialization of CTEs in queries.
-    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
\ No newline at end of file
+    This can also be specified on a per-query basis using the ``cte_partitioning_provider_catalog`` session property.
+
+
+Logging Properties
+------------------
+
+``log.path``
```

Comment:
@Akanksha-kedia Have we tested this config property? I tried to set this config but I am getting the below error

```
1) Error: Configuration property 'log.output-file' (=/Users/imjalpreet/Documents/PrestoRuntime/data/var/log/server.log) conflicts with property 'log.path' (=~/Downloads/server.log)

com.google.inject.ConfigurationException: Guice configuration errors:

1) Error: Configuration property 'log.output-file' (=/Users/imjalpreet/Documents/PrestoRuntime/data/var/log/server.log) conflicts with property 'log.path' (=~/Downloads/server.log)
```